packages/e2e-tests/src/config/bootstrap.js

```diff
@@ -154,12 +154,16 @@ function removePageEvents() {
 function observeConsoleLogging() {
   page.on('console', (message) => {
     const type = message.type();
-    if (!Object.hasOwnProperty.call(OBSERVED_CONSOLE_MESSAGE_TYPES, type)) {
-      return;
-    }
+    // if (!Object.hasOwnProperty.call(OBSERVED_CONSOLE_MESSAGE_TYPES, type)) {
+    //   return;
+    // }
 
     let text = message.text();
 
+    if (text.includes('Blocked attempt to create')) {
```

Comment:
This file is for the e2e tests, not karma

Reply:
ooof good catch